packages/modules/product/src/models/product-category.ts

```diff
@@ -112,6 +112,9 @@ class ProductCategory {
   @Property({ columnType: "timestamptz", nullable: true })
   deleted_at?: Date
 
+  @Property({ columnType: "jsonb", nullable: true })
+  metadata?: Record<string, unknown> | null
```

Comment:
```suggestion
  metadata: Record<string, unknown> | null = null
```

Reply:
You also need to add this to the types:

```
metadata?: Record<string, unknown> | null
```
 file: `https://github.com/medusajs/medusa/blob/develop/packages/core/types/src/http/product-category/common.ts#L5`

Reply:
Yeah we mentioned it in Slack but Oli merged already 😄